augraphy/augmentations/markup.py

```diff
@@ -190,7 +190,7 @@ def __call__(self, image, layer=None, force=False):
             # adding randomization.
             choice = random.choice([False, True])
             x, y, w, h = cv2.boundingRect(cnt)
-            if choice and (w > h * 2) and (w * h < (markup_mask.shape[0] * markup_mask.shape[1]) / 10):
+            if choice and (w > h * 2) and (w * h < (markup_mask.shape[0] * markup_mask.shape[1]) / 10) and h > 10:
```

Comment:
Added `h > 10` here to get contour with minimum height of 10. The character height should be at least 10 , else it will not be selected for markup effect.